modules/ROOT/pages/authentication.adoc

```diff
@@ -0,0 +1,21 @@
+= Configuring Authentication
+
+== Enabling SSH password authentication
+
+Fedora CoreOS disables SSH password authentication by default. To enable it, use the following Fedora CoreOS Config:
+
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  files:
+    - path: /etc/ssh/sshd_config.d/02-enable-passwords.conf
+      mode: 0644
+      contents:
+        inline: |
+          # Fedora CoreOS disables SSH password login by default.
+          # Enable it.
+          # This file must sort before 04-disable-passwords.conf.
```

Comment:
I wonder if since SSHD uses a "first one wins" strategy for config merging we should make the files we write easier to override by putting them at a higher number. i.e., `80-disable-passwords.conf`

Reply:
We can't.  The Fedora package ships an `05-redhat.conf` that includes `PasswordAuthentication yes`.

Reply:
ahh - grr. I guess my same question would apply to the rpm maintainers. Started a discussion there: https://src.fedoraproject.org/rpms/openssh/pull-request/9